src/Base32.h

```diff
@@ -45,7 +45,10 @@ inline std::string encode(const Data& val, const char* alphabet = nullptr) {
     }
     // perform the base32 encode
     char* retval = base32_encode(val.data(), inLen, buf, outLen, alphabet);
-    assert(retval != nullptr);
+    if (retval == NULL) {
```

Comment:
Why not nullptr here too instead of NULL?

Reply:
Thanks, it's a typo